ddtrace/tracer/option.go

```diff
@@ -138,6 +138,21 @@ type config struct {
 
 	// enabled reports whether tracing is enabled.
 	enabled bool
+
+	// postProcessor holds the function used to process finished spans of a trace.
+	// It reports whether the trace should be dropped.
+	postProcessor func([]ReadWriteSpan) bool
+}
+
+// WithPostProcessor enables processing finished spans of a trace by f. Spans
+// can be read from and written to in the processor by using the methods specified by
+// the ReadWriteSpan interface. f should return false if the trace should be dropped.
```

Comment:
```suggestion
// the ReadWriteSpan interface. f reports whether the trace should be dropped.
```
IIRC this was pretty common throughout the go repo: using the word "reports" when a boolean return value is involved.

Reply:
Thanks 🙇. Will make sure to use "reports" for a boolean return in the future!